frontend/README.md

```diff
@@ -1,5 +1,13 @@
-# Project created with React + TypeScript + Vite. Uses JSON Server to simulate an external Data Base
+## Project created with React + TypeScript + Vite.
 
-1. in project root directory run `npm i` to install all the dependencies
+### Configuration
 
-2. to start the project run `npm run dev` and **ctrl+click** to on the link to open in the browser
+1.  Create a .env file in the frontend folder and define the following variable:
+
+    ```env
+    VITE_PRODUCTS_DB_PORT=your_desired_port_number
```

Comment:
incorrect

Reply:
I missed that, is there a way to search entire project files for variable names and change them all at once?

Reply:
There is - to simply the search field after clicking the magnifying glass on the left side toolbar. I will try to use it now when renaming.